tests/fixtures/fixtures.py

```diff
@@ -369,8 +369,8 @@ def tiny_mistral_model_helper(config):
 
 
 @pytest.fixture(scope='session')
-def _session_tiny_t5_model(_session_tiny_t5_config):  # type: ignore
-    return tiny_t5_model_helper(_session_tiny_t5_config)
+def _session_tiny_mistral_model(_session_tiny_mistral_config):  # type: ignore
+    return tiny_mistral_model_helper(_session_tiny_mistral_config)
```

Comment:
whyd u kill t5?

Reply:
it was a typo from previous PR. there were two t5s and no mistral :)